bodhi/client/__init__.py

```diff
@@ -101,6 +103,21 @@ def _add_options(func):
     return _add_options
 
 
+def handle_errors(method):
+    """ A decorator that echoes neat error messages on AuthError or BodhiClientException. """
+    @functools.wraps(method)
+    def wrapper(*args, **kwargs):
+        try:
+            method(*args, **kwargs)
+        except AuthError as e:
+            click.echo("%s: Check your FAS username & password" % (e))
+            sys.exit(1)
+        except bindings.BodhiClientException as e:
+            click.echo(str(e))
+            sys.exit(-1)
```

Comment:
thanks! changed it to 2 in the test as well. The reason i set it to -1 was that the test was asserting the error code be -1, so i just assumed it was a valid exit code.